content/workshops/cascading-styles-deep.md

```diff
@@ -34,68 +34,107 @@ events:
-then this workshop is for you!
-We'll approach the language as a cohesive system,
-learn new features along the way,
-and then put it all in practice
-with a resilient approach to CSS architecture.
-
-Join Miriam for a deep dive
-into what makes the language work,
-and how we can harness its power
-to **develop resilient and delightful experiences**
-that hold up across browsers, languages, and device interfaces.
+{% import 'utility.macros.njk' as utility %}
+{% import 'contact.macros.njk' as contact %}
+{% import 'embed.macros.njk' as embed %}
+{% import 'quotes.macros.njk' as quotes %}
+
+## What Will Attendees Get?
+
+{{ embed.icon_block(
+  data=included,
+  grid='fit'
+) }}
+
+## Contact Us to Book a Workshop
+
+{{ contact.form(
+  submit='Book Now',
+  name='workshop',
+  extraActions='or [schedule a call to learn more »](https://calendly.com/oddbirdllc/schedule-a-workshop)'
+) }}
+
+{{ quotes.grid(press) }}
 
 ## What Will Attendees Learn In This Workshop?
 
-- Building maintainable CSS systems with progressive enhancement
-- The entire cascade, why it exists,
-  and what all is involved (much more than specificity!)
-- Managing style conflicts with cascade layers, scope, and nesting
-- Modern selectors like `:is()`, `:where()`, and `:has()`
-- Value resolution and validation, beyond cascading and inheritance
-- Using custom properties to build adaptive systems
-- The advantages of logical vs physical dimensions
-- The layout flow with intrinsic and extrinsic sizing
-- Size comparison functions like min(), max(), and clamp()
-- When to use flexbox vs grid for layout and alignment
-- Fast and dynamic layouts using subgrid and container queries
+- Building **maintainable CSS systems** with **progressive enhancement**
+- **The Cascade**, why it's there, and how to use it
+- **Managing conflicts** with cascade layers, scope, and nesting
+- **Modern selectors** like `:is()`, `:where()`, and `:has()`
+- Using **custom properties** to build adaptive systems
+- **CSS Layout** with grid and flexbox
+- Dynamic layouts using **subgrid** and **container queries**
 
 ## Who Is This For?
 
 This workshop is intended for designers & developers
-with at least a basic understanding in HTML and CSS.
+with at least a basic understanding of HTML and CSS.
 You don’t need to be an expert to keep up,
 but even the experts are likely to learn something new.
 
-## Schedule
+{{ utility.main_action(
+  content='Bring this workshop to your company »',
+  url='#contact-us-to-book-a-workshop'
+) }}
+
+## About Miriam
+
+{% set miriam = collections.birds | authorPage('miriam') %}
+{{ miriam.data.bio | md | safe }}
 
-The workshop is taught in five sessions, each with two parts:
+## Workshop Outline
+
+The workshop is taught in five sessions:
 
 ### 1. Resilient Styles: a Declarative Cascade
 
+{{ embed.figure([{
+  img: 'pages/css-workshop/cascade.jpg',
+  alt: 'Yoda reaching out to control the force, surrounded by jungle swamp'
+}]) }}
```

Comment:
Are these placeholder images, or are we planning to publish them?

Reply:
These are the images we used for the modules in the workshop site - they come from the slides. The plan was to publish them.